_emails/buffalo.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Buffalo, NY
+permalink: "/buffalo"
+name: Buffalo, NY Resident for Defunding the BPD
+state: NY
+city: Buffalo
+layout: email
+recipients:
+  - mayor@city-buffalo.com
+  - jferoleto@city-buffalo.com
+  - dpridgen@city-buffalo.com
+  - dfranczyk@city-buffalo.com
+  - mertha@city-buffalo.com
+  - jmontour@city-buffalo.com
+  - jpaul@city-buffalo.com
+  - vshaw@city-buffalo.com
+  - sgould@city-buffalo.com
+  - tstjean@city-buffalo.com
+  - wdoll@city-buffalo.com
+  - dthomas@city-buffalo.com
+  - choyt@city-buffalo.com
+  - mnowakowski@city-buffalo.com
+  - rfontana@city-buffalo.com
+  - uwingo@city-buffalo.com
+  - darivera@city-buffalo.com
+  - jgolombek@city-buffalo.com
+  - cscanlon@city-buffalo.com
+  - rwyatt@city-buffalo.com
+subject: Buffalo, NY Resident for Defunding the BPD
+body: "My name is [your name], and I am a resident of Buffalo. The incident as reported by WBFO (NPR) on June 4, 2020 is appalling and inhumane. This is a display of egregious police brutality. Please watch the video if you are not familiar: [video link](https://twitter.com/wbfo/status/1268712530358292484). This should speak for itself in justifying the immediate need to defund the BPD.\n\nSincerely,\n\n[your name]"
```

Comment:
per conversation on #69 , please enter placeholders ([your name], etc) in all caps (i.e. [YOUR NAME]) and enter placeholders for name, address, email, and phone number in the signature